integration-test.Rmd

```diff
@@ -95,6 +95,7 @@ ensure_empty_directory <- function(directory) {
   }
 
   dir_create(directory)
+  file_create(path(directory, ".gitkeep.txt"))
```

Comment:
1. Is the result the same as to not deleting .gitkeep.txt? 

I think .gitkeep.txt has some useful comments that explain why the file exist. But here 
we would end up with an empty file, right?

Reply:
Would it be better to instead not remove .gitkeep.txt? (See https://github.com/2DegreesInvesting/PACTA_analysis/pull/106/commits/f1786137e5f912474d7f2629b6cb45fd8dc882cd)